django_loci/static/django-loci/js/loci.js

```diff
@@ -207,7 +207,10 @@ django.jQuery(function ($) {
     function locationChange(e, initial) {
         function loadIndoor() {
             indoorForm();
-            if ($type.val() !== 'indoor') { return; }
+            if ($type.val() !== 'indoor') {
+                $indoor.hide();
```

Comment:
why is this necessary?

Reply:
While i was testing the work i did, I realized that if i selected a location of type `indoor`, the `indoor fields` showed up which was very good, but when i changed the type back to `outdoor`, i expected the `indoor fields` to be hidden again but that was not the case, reason while i added the `$indoor.hide()` when `type != 'indoor'` 
no
Actually it had no connection with the issue i was solving but i noticed it while i was testing and taught of doing of that.

Reply:
ok, I will double check this. Please follow up on the rest of the comments when you can, hopefully we'll be done with this soon :blush: